### ISSUE TITLE
Support decimal like partial title

### ISSUE THREAD

**Issue by Ravenclawer** [tgrosinger/crumbs-obsidian#2]
**Is your feature request related to a problem? Please describe.**

I have created the following notes
![image](https://user-images.githubusercontent.com/73892411/206609852-e46508a3-94b3-4d1b-8719-38a986e82bfd.png)
and the plugin could recognize the “000 title” “000.001 title”，but only for root note， the plugin could only recognize the “000” part, which makes it hard to navigate back to the root note. 

**Describe the solution you'd like**

I know the concept of this plugin is the put the whole title in the hierarchy, but in some occation like SCI paper, the title might be too long to make it practical to include the whole title of a note in to the tile of their sub notes, so I wonder if you can support the decimal like partial title to render the hierarchy.

**Comment by tgrosinger** [comment]
Just to make sure I understand your request, you are asking for "000.001 test1" to be considered a child of "000 test8"?

**Comment by Ravenclawer** [comment]
Yes, That's my intention or make it simple, allow part of title to be used in constructing hierarchy, in this case, the numbers before space is used. 

if without this feature, some of the titles may be too long to use in the hierarchy
